data/sql/updates/pending_db_world/rev_1707833862449013500.sql

```diff
@@ -0,0 +1,8 @@
+-- disables buff stacking for items 49856, 49857, 49858, 49859, 49861, 49860
+INSERT INTO `spell_group` (`id`, `spell_id`, `special_flag`) VALUES
```

Comment:
when you do an insert, it is important to do a downquery as well by deleting the entries before the insert.

so here you would want a DELETE FROM `spell_group` with id 2 and spell_ids in there as well